src/NuGet.Core/NuGet.Common/Errors/NuGetLogCode.cs

```diff
@@ -252,6 +252,11 @@ public enum NuGetLogCode
         /// </summary>
         NU1503 = 1503,
 
+        /// <summary>
+        /// Central package management is in use but there are multiple feeds configured without using package source mapping.
+        /// </summary>
+        NU1504 = 1504,
```

Comment:
Same log NU1504 used in https://github.com/NuGet/NuGet.Client/pull/4484

Reply:
I swear I looked at #4484 and it wasn't using NU1504...  I'll update my code

Reply:
Most likely he changed recently.
![image](https://user-images.githubusercontent.com/8766776/158634174-44141759-34a4-4b51-8632-55cb32fcc434.png)